install.sh

```diff
@@ -661,6 +673,11 @@ rm -rf /var/lib/rancher/k3s
 rm -rf /var/lib/kubelet
 rm -f ${BIN_DIR}/k3s
 rm -f ${KILLALL_K3S_SH}
+
+if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ]; then
```

Comment:
```suggestion
if type yum >/dev/null 2>&1; then
```

Reply:
I was trying to be a little more precise with this, I think there are other distros that may have `yum` installed. I guess just  this command is pretty blind and doesn't matter if it fails